e2e/cypress/integration/messaging/ctrl_cmd_k_find_gm_by_matching_name_spec.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***********************************************************  ****
+
+// Group: @messaging
+
+describe('Messaging', () => {
+    let testTeam;
+    let firstUser;
+    let secondUser;
+    let thirdUser;
+
+    before(() => {
+        // # Login as test user
+        cy.apiInitSetup().then(({team, user}) => {
```

Comment:
Just for readability, would recommend to keep names consistent,
firstUser = user1
secondUser = user2
thirdUser = user3

Reply:
Thanks Joseph, this does make more sense haha :)